source/PlayServicesResolver/src/PlayServicesResolver.cs

```diff
@@ -97,6 +97,15 @@ internal class DependencyState {
             public void WriteToFile() {
                 try {
                     Directory.CreateDirectory(Path.GetDirectoryName(DEPENDENCY_STATE_FILE));
+                    if (UnityEditor.VersionControl.Provider.enabled && UnityEditor.VersionControl.Provider.isActive &&
```

Comment:
Thanks again, since this code is the same boilerplate as the code in https://github.com/googlesamples/unity-jar-resolver/pull/256 could you add a static method to do this instead?

e.g

if (!CheckoutFile(DEPENDENCY_STATE_FILE)) {
  return;
}
...

Also, it would be useful to describe what went wrong if this fails.  At the moment it will silently fail which means that when we get a bug report that describes something not working we won't have any idea where it failed.